appd/model/role.py

```diff
@@ -0,0 +1,59 @@
+"""
+Model classes for AppDynamics REST API
+
+moduleauthor:: Srikar Achanta <srikar.achanta@appdynamics.com>
+
+"""
+
+from . import JsonObject, JsonList
+
+
+class Role(JsonObject):
+
+    FIELDS = {'id': '', 'name': '', 'description': ''}
+    # FIELDS = {'id': '', 'name': ''}
+
+    def __init__(self, id='0', name=None, description=None):
+        self.id, self.name, self.description = id, name, description
+
+
+class Roles(JsonList):
+    """
```

Comment:
Whitespace issue.

Reply:
I fixed the whitespace issue in role.py, group.py and user.py classes. do you want me to send another pull request, or wait and send these changes in my next pull request?

This is how I have it now:

class Users(JsonList):
    """
    Represents a collection of :class:User objects. Extends :class:UserList, so it supports the
    standard array index and :keyword:`for` semantics.
    """

    def __init__(self, initial_list=None):
        super(Users, self).__init__(User, initial_list)

    def __getitem__(self, i):

Reply:
webpage is not showing the spaces correctly. In the code snippet pasted above, comments section is in one line and each line has 4 spaces. Github is throwing an error, while trying to upload the file.

Reply:
I believe you can update this pull request with the changes.